### PR TITLE
Applies new checkstyle to OVM3 module

### PR DESCRIPTION
**Pull request by wilderrodrigues**
The old checkstyle has already been removed and the module is now compliant with the new checkstyle.

### CODE REVIEW

**Comment by wilderrodrigues** [comment]
Ping @remibergsma @borisroman 

Tests
- Build source including tests

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 09:34 min
[INFO] Finished at: 2016-01-28T00:44:11+01:00
[INFO] Final Memory: 95M/570M
[INFO] ------------------------------------------------------------------------
```

Will do some more before before asking for a merge.

**Comment by wilderrodrigues** [comment]
Please, hold on this. It built locally, but failed in the bubble because a file for some reason was not pushed correctly:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7:23.872s
[INFO] Finished at: Thu Jan 28 00:02:27 GMT 2016   
[INFO] Final Memory: 59M/292M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.2:compile (default-compile) on project cloud-plugin-hypervisor-ovm3: Compilation failure
[ERROR] /data/git/cs2/cosmic/cosmic-plugin-hypervisor-ovm3/src/main/java/com/cloud/hypervisor/ovm3/objects/PoolOCFS2.java:[26,7] error: class PoolOcfS2 is public, should be declared in 
a file named PoolOcfS2.java
```

The file on my mac:

```
sbpltk1zffh04:cosmic-plugin-hypervisor-ovm3 wrodrigues$ ls -lart ./src/main/java/com/cloud/hypervisor/ovm3/objects/PoolOcfS2.java
-rw-r--r--  1 wrodrigues  SBP\Domain Users  4727 Jan 27 23:37 ./src/main/java/com/cloud/hypervisor/ovm3/objects/PoolOcfS2.java
```

But the file on github:

image

**Comment by wilderrodrigues** [comment]
Ping @borisroman @remibergsma @miguelaferreira 

Fixed the file name issue.

```
INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 8:27.752s
[INFO] Finished at: Thu Jan 28 06:55:20 GMT 2016
[INFO] Final Memory: 80M/417M
[INFO] ------------------------------------------------------------------------
[root@cs2 cosmic]# 
```

Will deploy DC; Networks; and VMs

**Comment by wilderrodrigues** [comment]
- Built RPMs from source
- Deployed DC
- Deployed isolated network and VM

Will run some integration tests.

File: pom.xml [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-
-  Unless required by applicable law or agreed to in writing,
-  software distributed under the License is distributed on an
-  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-  KIND, either express or implied. See the License for the
-  specific language governing permissions and limitations
-  under the License.
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
```
Comment (borisroman) [inline pom.xml:1]: Maybe remove the license header entirely?

**Comment by wilderrodrigues** [comment]
Some tests executed:

```
[root@cs2 integration]# less /tmp//MarvinLogs/test_vm_life_cycle_SX1HJV/results.txt 
Test router internal advanced zone ... === TestName: test_02_router_internal_adv | Status : SUCCESS ===
ok
Test restart network ... === TestName: test_03_restart_network_cleanup | Status : SUCCESS ===
ok
Test router basic setup ... === TestName: test_05_router_basic | Status : SUCCESS ===
ok
Test router advanced setup ... === TestName: test_06_router_advanced | Status : SUCCESS ===
ok
Test stop router ... === TestName: test_07_stop_router | Status : SUCCESS ===
ok
Test start router ... === TestName: test_08_start_router | Status : SUCCESS ===
ok
Test reboot router ... === TestName: test_09_reboot_router | Status : SUCCESS ===
ok
Test reset virtual machine on reboot ... === TestName: test_01_reset_vm_on_reboot | Status : SUCCESS ===
ok
Test advanced zone virtual router ... === TestName: test_advZoneVirtualRouter | Status : SUCCESS ===
ok
Test Deploy Virtual Machine ... === TestName: test_deploy_vm | Status : SUCCESS ===
ok
Test Multiple Deploy Virtual Machine ... === TestName: test_deploy_vm_multiple | Status : SUCCESS ===
ok
Test Stop Virtual Machine ... === TestName: test_01_stop_vm | Status : SUCCESS ===
ok
Test Start Virtual Machine ... === TestName: test_02_start_vm | Status : SUCCESS ===
ok
Test Reboot Virtual Machine ... === TestName: test_03_reboot_vm | Status : SUCCESS ===
ok
Test destroy Virtual Machine ... === TestName: test_06_destroy_vm | Status : SUCCESS ===
ok
Test recover Virtual Machine ... === TestName: test_07_restore_vm | Status : SUCCESS ===
ok
Test migrate VM ... SKIP: At least two hosts should be present in the zone for migration
Test destroy(expunge) Virtual Machine ... === TestName: test_09_expunge_vm | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 18 tests in 2772.219s

OK (SKIP=1)
/tmp//MarvinLogs/test_vm_life_cycle_SX1HJV/results.txt (END)
```

File: pom.xml (hunk 1 above)
Comment (wilderrodrigues) [inline pom.xml:1]: Should we perhaps remove all the headers from all files - Java, xml, etc - in one go and different PR?

Comment (borisroman) [inline pom.xml:1]: Okay!

**Comment by wilderrodrigues** [comment]
Ping @borisroman @remibergsma 

Can we get this one merged? :)

**Comment by borisroman** [comment]
LGTM :+1:
